connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectorHandle.java

```diff
@@ -284,7 +283,7 @@ public StartAndStopLatch expectedStarts(int expectedStarts, boolean includeTasks
                 ? taskHandles.values().stream()
                 .map(task -> task.expectedStarts(expectedStarts))
                 .collect(Collectors.toList())
```

Comment:
```suggestion
                .toList()
```

Could you check other `collect(Collectors.toList())` usage and change to ` .toList()`